docs/website/docs/general-usage/customising-pipelines/removing_columns.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Removing columns
+description: Removing columns by passing list of column names
+keywords: [deleting, removing, columns, drop]
+---
+
+# Removing columns
+
+Removing columns before loading data into a database is a reliable method to eliminate sensitive or
+unnecessary fields. For example, in the given scenario, a source is created with a "country_id" column,
+which is then excluded from the database before loading.
+
+Let's create a sample pipeline demonstrating the process of removing a column.
+
+1. Create a source function that creates dummy data as follows:
+
+   ```python
+   import dlt
+
+   @dlt.source
+   def dummy_source(prefix: str = None):
+       #This function creates a dummy data source.
+       @dlt.resource(write_disposition='replace')
+       def dummy_data():
+             for _ in range(3):
+               yield {'id': _, 'name': f'Jane Washington {_}', 'country_code': 40 + _}
+       return dummy_data()
+   ```
+   This function creates three columns `id`, `name` and `country_code`.
+
+1. Next, create a function to filter out columns from the data before loading it into a database as follows:
+
+   ```python
+   def remove_columns(doc, remove_columns=None):
+
+       if remove_columns is None:
+           remove_columns = []
+
+       # Iterating over the list of columns to be removed
+       for column_name in remove_columns:
+           # Removing the column if it exists in the document
+           if column_name in doc:
+               del doc[column_name]
+
+       return doc
+   ```
+
+   `doc`: The document (row) from which columns will be removed.
+
+   `remove_columns`: List of column names to be removed, defaults to None.
+
+1. Next, declare the columns to be removed from the table, and then modify the source as follows:
+
+   ```python
+   # Example columns to remove:
+   remove_columns_list = ["country_code"]
+
+   # 1. Create an instance of the source so you can edit it.
```

Comment:
delete numeration in comments, it's confusing

Reply:
deleted